lua/metals.lua

```diff
@@ -182,4 +188,32 @@ M['metals/status'] = function(_, _, params)
   end
 end
 
+M['metals/publishDecorations'] = function(err, _, decorations)
+  if err then
+    print("metals.publishDecorations: Server error:", err)
+  end
+  if not decorations then
+    return
+  end
+
+  local uri = decorations.uri
+  local bufnr = vim.uri_to_bufnr(uri)
+  if not bufnr then
+    print("metals.publishDecorations: Couldn't find buffer for ", uri)
```

Comment:
Do you know if there is a way to color these? It might be good to get an `error`, `warning`, `info` in here to follow what the LSP log messages do. This doesn't need to be done for this, but just a note.